config/swagger-ui.php

```diff
@@ -42,4 +42,26 @@
         'client_id' => env('SWAGGER_UI_OAUTH_CLIENT_ID'),
         'client_secret' => env('SWAGGER_UI_OAUTH_CLIENT_SECRET'),
     ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Swagger UI - Current Server
+    |--------------------------------------------------------------------------
+    |
+    | This allows you to use all servers url in your openapi.json file (Not only local server)
+    | Set this option to false, to use list of servers from openapi.json file
+    |
+    */
+    'local_server_only' => env('SWAGGER_LOCAL_SERVER_ONLY', true),
+
+    /*
+    |--------------------------------------------------------------------------
+    | Swagger UI - Api base path
+    |--------------------------------------------------------------------------
+    |
+    | This allows you to set suffix to you APP_URL
+    | Set this option to /api/v1, to send your requests to APP_URL/api/v1/... endpoints
+    |
+    */
+    'api_base_path' => env('SWAGGER_API_BASE_PATH'),
```

Comment:
What is the reasoning behind this? Should your API routes in your OpenAPI file not already contain the API base path?

Reply:
Yes, I have it in openapi file, https://example.com/api/v1
but it replaced in 
{code}
$json['servers'] = [
            ['url' => config('app.url')],
        ];
{code}
because my app.url is https://example.com

with this option we recreate original url from openapi file with 
{code}
$json['servers'] = [
            ['url' => config('app.url').config('swagger-ui.api_base_path')],
        ];
{code}